### ISSUE TITLE
How to create a $text index

### ISSUE THREAD

**Issue by jaime-franco** [balderdashy/sails-mongo#357]
Hi,

Is there a way to create a mongodb $text index from sails, for the moment i have to do it manually.

**Comment by sailsbot** [comment]
Thanks for posting, @jaime-franco.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!

**Comment by jaime-franco** [comment]
If someone is looking for this, for create a $text index automatically i think is not possible but you can use native query for create the index.
```
Model.native(function(err,collection){
		collection.createIndex({field : "text"},function(err,result){
			//Your logic for handle the result
		});
	});	
```

**Comment by gil0mendes** [comment]
Thanks, I really need that now. But the Waterline should have a way to do that without this ugly workaround. 😕
